templates/apps/sumo-aws-apps/firewall-manager-org/firewall-manager-org-waf-network-firewall-policy.yaml

```diff
@@ -206,9 +204,7 @@ Resources:
                 - ''
             
 
-  rFmsCloudFrontDefaultPolicy:
-    DependsOn:
-      - KinesisFirehoseDeliveryStream    
```

Comment:
these policies needs KinesisFirehoseDeliveryStreamArn so it should exists before we create these policies

Reply:
if Create Delivery Stream then the LoggingConfiguration will need KinesisFirehoseDeliveryStreamArn. In case Create Delivery Stream is not used, LoggingConfiguration will not be configured